cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -308,7 +307,7 @@ pluginConfig:
 		{
 			name:          "invalid config file in componentconfig/v1alpha2",
 			options:       &Options{ConfigFile: invalidconfigFile},
-			expectedError: "no kind \"KubeSchedulerConfiguration\" is registered for version \"componentconfig/v1alpha2\"",
```

Comment:
why is this test passing now? don't we have a registered v1alpha2 version (i.e., isn't this version valid right now?)

Reply:
Because it said `componentconfig/v1alpha2`, instead of `kubescheduler.config.k8s.io/v1alpha2`. I also thought that there was a bug.

Reply:
oh, ok :)